packages/cicero-core/src/templatesaver.js

```diff
@@ -42,6 +45,14 @@ class TemplateSaver {
 
         let zip = new JSZip();
 
+        let templateSignatures = {
+            signatures: {
```

Comment:
Why isn't the `signatureObject` containing all the signatures? Where does it come from?